cloudsync/sync/manager.py

```diff
@@ -1304,22 +1304,22 @@ def conflict_rename(self, side, path):
                 conflict_path = self.providers[side].join(folder, conflict_name)
                 new_oid = self.providers[side].rename(oinfo.oid, conflict_path)
             except ex.CloudFileExistsError:
-                log.debug("already exists %s", conflict_name)
+                log.info("already exists %s", conflict_name)
                 i = i + 1
                 conflict_name = base + ".conflicted" + str(i) + ext
 
-        log.debug("conflict renamed: %s -> %s", path, conflict_path)
+        log.info("conflict renamed: %s -> %s", path, conflict_path)
         return oinfo.oid, new_oid, conflict_path
 
     def embrace_change(self, sync, changed, synced):  # pylint: disable=too-many-return-statements, too-many-branches
         if sync[changed].path or (sync[changed].exists == EXISTS):
             translated_path = self.translate(synced, sync[changed].path)
             if not translated_path:
                 if sync[changed].sync_path:  # This entry was relevent, but now it is irrelevant
-                    log.debug(">>>Removing remnants of file moved out of cloud root")
+                    log.info(">>>Removing remnants of file moved out of cloud root")
                     sync[changed].exists = TRASHED  # This will discard the ent later
                 else:  # we don't have a new or old translated path... just irrelevant so discard
-                    log.log(TRACE, ">>>Not a cloud path %s, ignoring", sync[changed].path)
+                    log.info(">>>Not a cloud path %s, ignoring", sync[changed].path)
```

Comment:
This line looks like it will get triggered every time a file outside of the bed gets changed. Is that accurate?

Reply:
Ya I think you might be right